CorinneTest/Person.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class Person 
+    {
+        private string name;
+        public string Name
+        {
+            get => this.name; 
+            set { this.name = value; }
+        }
+
+        private int? age;
+        public int? Age
+        {
+            get => this.age;
+            set {
+                if (value < 1) value = 1;
+                else if (value > 100) value = 100;
+                this.age = value;
+            }
+        }
+
+        private Gender gender;
```

Comment:
using the word private isn't necessary since it is the default access of a variable.  i know that its a bit of a java holdover in your coding style, but the Static Analysis (an algorithm that checks your code and grades it) would complain about using "private"

Reply:
got it